openslides/motions/access_permissions.py

```diff
@@ -49,7 +49,8 @@ def get_restricted_data(
                     is_submitter = False
 
                 # Check see permission for this motion.
-                required_permission_to_see = full['state_required_permission_to_see']
+                from .models import State  # import it here to resolve circular imports
+                required_permission_to_see = State.objects.get(pk=full['state_id']).required_permission_to_see
```

Comment:
I see: This is a problem. I think we do not want to have extra DB queries here. @ostcar What do you think?

Reply:
@normanjaeckel @ostcar ping

Reply:
I think we do not want to have extra DB queries here.